span.go

```diff
@@ -60,21 +73,38 @@ func (e *traceExporter) convertSpan(s *trace.SpanData) *ddSpan {
 	if s.ParentSpanID != (trace.SpanID{}) {
 		span.ParentID = binary.BigEndian.Uint64(s.ParentSpanID[:])
 	}
+
+	httpCode := httpCode(s.Status.Code)
```

Comment:
Can we call this simply `code`? Then, we can just do:
```go
code, ok := statusCodes[s.Status.Code]
if !ok {
    code = codeDetails{
        message: "ERR_CODE_" + strconv.FormatInt(int64(s.Status.Code), 10),
        status: http.StatusInternalServerError,
    }
}
```
Then we can just use `code.message` and `code.status` everywhere.

Or, if you really prefer to have a function, we could just have it return `codeDetails` directly, no need to have two separate functions.